clients/java/dkv-client/src/test/resources/Procfile

```diff
@@ -1,28 +1,28 @@
 # run using `goreman -exit-on-error start`
 
-dkv_discovery1:./bin/dkvsrv -role discovery -listen-addr 127.0.0.1:8001 -db-folder /tmp/dkvsrv/d0/data -discovery-service-config internal/discovery/discovery.ini -nexus-node-url "http://127.0.0.1:7021" -nexus-cluster-url "http://127.0.0.1:7021,http://127.0.0.1:7022,http://127.0.0.1:7023" -nexus-log-dir /tmp/dkvsrv/d0/logs -nexus-snap-dir /tmp/dkvsrv/d0/snap -nexus-snapshot-count 100 -nexus-snapshot-catchup-entries 5
-dkv_discovery2:./bin/dkvsrv -role discovery -listen-addr 127.0.0.1:8002 -db-folder /tmp/dkvsrv/d1/data -discovery-service-config internal/discovery/discovery.ini -nexus-node-url "http://127.0.0.1:7022" -nexus-cluster-url "http://127.0.0.1:7021,http://127.0.0.1:7022,http://127.0.0.1:7023" -nexus-log-dir /tmp/dkvsrv/d1/logs -nexus-snap-dir /tmp/dkvsrv/d1/snap -nexus-snapshot-count 100 -nexus-snapshot-catchup-entries 5
-dkv_discovery3:./bin/dkvsrv -role discovery -listen-addr 127.0.0.1:8003 -db-folder /tmp/dkvsrv/d2/data -discovery-service-config internal/discovery/discovery.ini -nexus-node-url "http://127.0.0.1:7023" -nexus-cluster-url "http://127.0.0.1:7021,http://127.0.0.1:7022,http://127.0.0.1:7023" -nexus-log-dir /tmp/dkvsrv/d2/logs -nexus-snap-dir /tmp/dkvsrv/d2/snap -nexus-snapshot-count 100 -nexus-snapshot-catchup-entries 5
+dkv_discovery1:./bin/dkvsrv --role discovery --listen-addr 127.0.0.1:8001 --node-name d1 --nexus-node-url "http://127.0.0.1:7021" --nexus-cluster-url "http://127.0.0.1:7021,http://127.0.0.1:7022,http://127.0.0.1:7023" --config clients/java/dkv-client/src/test/resources/dkv_config.yaml 
```

Comment:
This is a specific feature that we wanted in our profile, that we should be able to override any config via arguments. This will allow having complex profile without need for so many config files.

Reply:
Fair point.